onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -194,6 +194,10 @@ def quantize_static(model_input,
                                               inserts both QuantizeLinear/DeQuantizeLinear nodes to weight.
             OpTypesToExcludeOutputQuantizatioin = list of op type : Default is []. If any op type is specified, it won't quantize  
                                                                     the output of ops with this specific op types.
```

Comment:
as TRT has so many , i'm thinking we may need a config file for different execution provider